openedx/core/lib/exceptions.py

```diff
@@ -10,3 +10,8 @@ class CourseNotFoundError(ObjectDoesNotExist):
 class PageNotFoundError(ObjectDoesNotExist):
     """ Page was not found. Used for paginated endpoint. """
     pass
+
+
+class DiscussionNotFoundError(ObjectDoesNotExist):
+    """ Discussion Module was not found. """
+    pass
```

Comment:
Nit: I see you're following the style of the rest of the docstrings in this file, but the extra spaces hurt my brain.  Can you switch to the three-line docstring style?